### PR TITLE
Feature/update/plotly 2.18.1

### PR DESCRIPTION
**Pull request by mmichal9**

### CODE REVIEW

File: offline/plot.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func computeOptions(def Options, opt ...Options) Options {
 
 var baseHtml = `
 	<head>
-		<script src="https://cdn.plot.ly/plotly-1.58.4.min.js"></script>
+	    <script src="https://cdn.plot.ly/plotly-2.18.1.min.js"></script>
```
Comment (MetalBlueberry) [inline offline/plot.go:83]: For some reason I don't understand, when I run `offline.Show` it doesn't work on Google Chrome but it works on Firefox after updating to 2.18.1 :confused: 

how did you test this change?

File: graph_objects/layout_gen.go [hunk 2]
```diff
@@ -42,9 +42,9 @@ type Layout struct {
 	Bargroupgap float64 `json:"bargroupgap,omitempty"`
 
 	// Barmode
-	// default: stack
+	// default: group
 	// type: enumerated
-	// Determines how bars at the same location coordinate are displayed on the graph. With *stack*, the bars are stacked on top of one another With *overlay*, the bars are plotted over one another, you might need to an *opacity* to see multiple bars.
+	// Determines how bars at the same location coordinate are displayed on the graph. With *stack*, the bars are stacked on top of one another With *relative*, the bars are stacked on top of one another, with negative values below the axis, positive values above With *group*, the bars are plotted next to one another centered around the shared location. With *overlay*, the bars are plotted over one another, you might need to an *opacity* to see multiple bars.
```
Comment (MetalBlueberry) [inline graph_objects/layout_gen.go:47]: How did you change this file? it should be generated automatically from the schema and I don't see schema changes on this PR

**Review by MetalBlueberry — CHANGES_REQUESTED** [review]
Than you very much for this PR :bow: 
I just left a few comments to clarify some things

File: offline/plot.go (hunk 1 above)
Comment (mmichal9) [reply offline/plot.go:83]: Sorry for the delay, I must have missed your message. 
Interesting, it is running on my Chrome just fine. Version 112.0.5615.49 (Official Build) (x86_64)

File: graph_objects/layout_gen.go (hunk 2 above)
Comment (mmichal9) [reply graph_objects/layout_gen.go:47]: Made a mistake somewhere. The schema update didnt actually work at the end so the [update: ploty to 2.18.1](https://github.com/MetalBlueberry/go-plotly/pull/10/commits/1d8c908fb6c06eb36f364f146610ed1d24e79b79) commit does nothing really. When I tried getting latest schema today it failed generation. I need to look into this a bit further.

**Comment by mmichal9** [comment]
@MetalBlueberry I think it will be best to close this PR. I can make a new one with just and example added and addressed schema update in a separate PR since it might the some more effort figure it out.

**Comment by MetalBlueberry** [comment]
Yes please, feel free to close this one and open individual PRs :smile: 

Thank you for contributing!! It really makes me happy :heart:

**Comment by MetalBlueberry** [comment]
Inactive PR

**Pull request closed without merging**